src/stable.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
Will need to clean this up.

Reply:
Perhaps yeah, I personally find the soup of `#[cfg]` on imports somewhat unappealing and am generally ok with dead code otherwise, so I might personally lean towards leaving this